easybuild/easyblocks/t/trinity.py

```diff
@@ -54,6 +54,7 @@ def __init__(self, *args, **kwargs):
         EasyBlock.__init__(self, *args, **kwargs)
 
         self.build_in_installdir = True
+        self.path = ''
```

Comment:
well, you either want it initialised with a value that's not going to give errors later on, or you can just skip initialising it, then it will also give errors later on. If you're going to initialise with None you're just making pylint happy, and nothing else, because you will end up getting errors since not is not what you expect to be in there.